src/app/lib/globalStyles/index.jsx

```diff
@@ -19,6 +19,12 @@ const GlobalStyleComponent = createGlobalStyle`
       box-sizing: inherit;
     }
 
+    i {
```

Comment:
Do we need this? the new font styles work, thats yet to be pulled into simorgh, ensures that persian cant be italic I think https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/__snapshots__/font-styles.test.js.snap#L59-L65

Reply:
@sadick254 is above right?

Reply:
@dr3 Yes thats right, `persian` doesn't have `boldItalic` defined and it should fall back to just `bold`